keychain-sdk/config.go

```diff
@@ -58,3 +49,25 @@ type Config struct {
 	// failed (but the blockchain might still include in a block later).
 	TxTimeout time.Duration
 }
+
+// Config is the configuration for the Keychain.
+type Config struct {
+	BasicConfig
+
+	// ConsensusNodeThreshold represents the number of nodes required to execute a pending key/sign request.
+	ConsensusNodeThreshold uint8
+
+	// GRPCURLs is the list of URLs of the gRPC server to connect to.
+	// e.g. "localhost:9090"
+	GRPCConfigs []GrpcNodeConfig
```

Comment:
since we call them "nodes" (Consensus**Node**Threshold represents the number of **nodes**...) we should be consistent. Also, the doc comment for GRPCConfigs is outdated.

```suggestion
	// ConsensusNodeThreshold represents the number of nodes required to execute a pending key/sign request.
	ConsensusNodeThreshold uint8

	// Nodes is the list of gRPC nodes to connect to for fetching incoming requests and broadcasting transactions.
	Nodes []GrpcNodeConfig
```